clr.nativebuild.props

```diff
@@ -0,0 +1,7 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <CL_MPCount>$(NumberOfEnabledCore)</CL_MPCount>
```

Comment:
Unfortunately I don't know enough about these properties to say this is the best way or not. I will see if I can find a contact in C++ land to review this change as well. Although please do not block on that if this has improvements then lets do it.